script.js

```diff
@@ -14,3 +14,225 @@ Items.forEach((n) => {
     dropdownMenu.classList.toggle('hide');
   });
 });
+
+// popup!!!
+
+const body = document.querySelector("body");
+const section = document.createElement("section");
+body.appendChild(section);
+section.innerHTML = `
+  <div class="popup-window pophide">
+  <div class="popup pophide">
+  <div class="phead">
+   <span class="xsign"><i class="fa-solid fa-xmark xsign-icon"></i></span>
+   <img src="" alt="" class="pop-img" />
+  </div>
+  <h2 class="phead-h2"></h2>
+    <ul class="plist">
+      <li><a href="#" class="panc card-a4">Ruby on rails</a></li>
+      <li><a href="#" class="panc card-a2">CSS</a></li>
+      <li><a href="#" class="panc card-a3">Javscript</a></li>
+    </ul>
+  <p class="ptext"> </p>
+  <ul class="pbuttons">
+  <button class="see" type="button" name="button">
+  <span>See Live</span>
+  <span><img src="images/upload.svg" alt=""></span>
+  </button>
+  <button class="see" type="button" name="button">
+  <span>See Source</span>
+  <span><img src="images/vector.svg" alt=""></span>
+  </button>
```

Comment:
https://github.com/danielmatama/matama-s-portfolio/blob/ce1cd0cfae27de6c4d3b9f02d9b3c4be40dd55a7/index.html#L83-L185



- [OPTIONAL] In this particular project, the section of the whole project along with the models should be created dynamically with Javascript. I can see that Modal popup data is populated with the data from the `projectDetails` array good job! 👏🏻 , but the base projects section is still not being created with Javascript.  Please use Javascript to dynamically render your projects as you did with the Modals or Popup windows pages.

- This is what the project requirements ask for:

![image](https://user-images.githubusercontent.com/83596135/189400760-cec4f52c-ab6e-4962-b57a-e6f1f1ee0b49.png)

- Here is an **[example](https://github.com/olawale-o/olawale-o.me/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95)**
- That way, adding projects in the future will be easy. You will only need to add it in one place - as another object in the `projectDetails` array. 😉 👍🏻 
 
 - (Note: the works section must be removed from HTML and generated with javascript )


## Note: these are only optional because due to oversight the previous reviewer probably missed them, However, These are so important.